docs/architecture.rst

```diff
@@ -264,3 +276,35 @@ output:
    :alt: Keeper state machine
 
    Keeper State Machine
+
+
+Multi-nodes architecture
+========================
+
+pg_auto_failover also allows to have more than one standby node, and
+implements the following three parameters to control the behavior of the
+system in case of a failover:
+
+  - number_sync_stanbys
```

Comment:
I guess an explanation similar to [here](https://github.com/citusdata/pg_auto_failover/pull/326#issue-445517663) might be useful.

In other words, it could be nice to explain what users can achieve by changing these values? I'm OK to skip it on github, but  we should probably have  it on docs repo  at  least?

Finally, it might be helpful to mention about the checks in  `FormationNumSyncStandbyIsValid ` function? Otherwise,  thinking about `number_sync_stanbys ` and `replication quorum` at  the same time  could be slightly hard for new users to comprehend?

Reply:
Added now.